ibis/backends/tests/test_generic.py

```diff
@@ -995,3 +995,93 @@ def test_pivot_longer(backend):
     )
     df = res.limit(5).execute()
     assert not df.empty
+
```

Comment:
Should this test be in test_table.py, next to the other test function? It is a function on a table, not on a generic expression?